pkg/util/pointer.go

```diff
@@ -10,6 +10,12 @@ func UInt64Ptr(v uint64) *uint64 {
 	return &v
 }
 
+// UInt32Ptr returns a pointer from a value. It will allocate a new heap object for it.
+func UInt32Ptr(v int64) *uint32 {
```

Comment:
Should this really receive a signed int instead of an unsigned int?

Reply:
Do you see a problem with this?